content/automate/ManageIQ/System/Event/MiqEvent/POLICY.class/__methods__/check_policy_prevent.rb

```diff
@@ -0,0 +1,8 @@
+#
+# Description: This method stops the retirement if the policy prevents
+#
+event = $evm.root['event_stream']
```

Comment:
@lfu 
Can this method be converted to the new style with namespaces.
e.g. https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/AutomationManagement/AnsibleTower/Operations/Methods.class/__methods__/available_credentials.rb
Also can we add a spec for this.